### DISCUSSION TITLE
Problem of registering into github edu

### DISCUSSION THREAD

**Discussion by Nhat-School** [community/community#124996] [New to GitHub]
We're still processing your application. Once this is complete you will have access to the student developer pack offers.
this message is remain for a long time (more than 2 month i think ).
Can anyone help me fix this error. Although i have shot my student card very carefully.

**Comment by davevad93** [comment]
Hi @Nhat-School ,

The processing time may vary due to different factors, although it is true that 2 months is a long time, so I would advise you to contact GitHub Support by creating a ticket [here](https://support.github.com/request/education).

**Comment by ArmSershant** [comment]
Hello @Nhat-School . The request can take anywhere from one hour to 165 days, I've created github edu request today and received a message that they will answer until November.
